### PR TITLE
Feature/472

### PR DESCRIPTION
**Pull request by RomanGalochkin**
https://github.com/foretold-app/foretold/issues/463
https://github.com/foretold-app/foretold/issues/472
https://github.com/foretold-app/foretold/issues/467

### CODE REVIEW

File: packages/server/config/temapltes.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+module.exports = [
```
Comment (OAGr) [inline packages/server/config/temapltes.js]: spelling error: server/config/*templates*.js

File: packages/server/config/temapltes.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      "item": "{{{ measurable.name }}}",
+      "description": "{{ agent.name }} has marked this question as unresolved",
+      "measurableId": "{{{ measurable.id }}}"
+    }
+  },
+  {
+    "name": "MEASURABLE_REACHED_RESOLUTION_DATE_FEED_ITEM",
+    "envelopeTemplate": {
+      "item": "{{{ measurable.name }}}",
+      "description": "{{ measurable.name }} has reached it's expected resolution date",
```
Comment (OAGr) [inline packages/server/config/temapltes.js]: why only two {{ here?

File: packages/server/src/async/producers/producer.js [hunk 3]
```diff
@@ -87,10 +90,12 @@ class Producer {
 
 Producer.data = data;
 Producer.TEMPLATE_NAME = TEMPLATE_NAME;
-Producer.TEMPLATE_NAME = TEMPLATE_NAME;
 Producer.NOTIFICATION_TYPE = NOTIFICATION_TYPE;
 Producer.EmailEnvelope = EmailEnvelope;
+
 Producer.FeedItem = FeedItem;
+Producer.FeedItemCommon = FeedItemCommon;
```
Comment (OAGr) [inline packages/server/src/async/producers/producer.js]: maybe ``FeedItemGeneric``?

File: packages/server/src/models-abstract/template-model.js [hunk 4]
```diff
@@ -13,6 +16,23 @@ class TemplateModel extends ModelPostgres {
     });
   }
 
+
+  /**
+   * It is a temporary solution.
```
Comment (OAGr) [inline packages/server/src/models-abstract/template-model.js]: Is there any story for a better solution, by chance?

File: packages/server/src/async/producers/producer.js (hunk 3 above)
Comment (RomanGalochkin) [reply packages/server/src/async/producers/producer.js]: Will bring many changes, should rename?

File: packages/server/config/temapltes.js (hunk 2 above)
Comment (RomanGalochkin) [reply packages/server/config/temapltes.js]: Yes, thanks, will update

File: packages/server/src/models/classes/feed-item-common.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  getDescription() {
+    return this.description;
+  }
+
+  /**
+   * @public
+   * @param {object} replacements
+   * @return {FeedItem}
+   */
+  mutate(replacements) {
```
Comment (OAGr) [inline packages/server/src/models/classes/feed-item-common.js]: ``mutate`` is a very generic name, like ``change``.  Please make more specific.

File: packages/server/src/models-abstract/template-model.js (hunk 4 above)
Comment (RomanGalochkin) [reply packages/server/src/models-abstract/template-model.js]: Need to think

**Comment by OAGr** [comment]
I left some minor comments, please review

**Comment by RomanGalochkin** [comment]
Sure

File: packages/server/src/async/producers/producer.js (hunk 3 above)
Comment (OAGr) [reply packages/server/src/async/producers/producer.js]: sure

File: packages/server/config/temapltes.js (hunk 1 above)
Comment (RomanGalochkin) [reply packages/server/config/temapltes.js]: Fixed.

File: packages/server/config/temapltes.js (hunk 2 above)
Comment (RomanGalochkin) [reply packages/server/config/temapltes.js]: Fixed.

File: packages/server/src/models/classes/feed-item-common.js (hunk 5 above)
Comment (RomanGalochkin) [reply packages/server/src/models/classes/feed-item-common.js]: Fixed.

File: packages/server/src/async/producers/producer.js (hunk 3 above)
Comment (RomanGalochkin) [reply packages/server/src/async/producers/producer.js]: Thanks, fixed.

**Comment by RomanGalochkin** [comment]
@OAGr Have updated, thanks for your tips, useful for me. Especially with understanding the difference between "common" and "generic" )

**Pull request merged**